Makefile

```diff
@@ -88,7 +89,11 @@ $(BUILD_DIR):
 	mkdir -p $(BUILD_DIR)
 
 $(BUILD_DIR)/$(PROJECT): $(BUILD_DIR) $(BUILD_FILES)
+ifneq ($(shell uname -s), Darwin)
 	$(GO) build -ldflags '$(LDFLAGS)' -tags '$(BUILDTAGS)' -o $@ ./cmd/security-profiles-operator
+else
+	$(GO) build -ldflags '$(LDFLAGS)' -tags '$(BUILDTAGSOSX)' -o $@ ./cmd/security-profiles-operator
+endif
```

Comment:
I'd prefer that we just set the `BUILDTAGS` differently when running on macOS. This way we can avoid code duplication.

Reply:
Agreed. Thanks